toolchain/check/check.cpp

```diff
@@ -61,26 +61,31 @@ static auto RenderImportKey(ImportKey import_key) -> std::string {
 static auto TrackImport(Map<ImportKey, UnitAndImports*>& api_map,
                         Map<ImportKey, Parse::NodeId>* explicit_import_map,
                         UnitAndImports& unit_info,
-                        Parse::Tree::PackagingNames import) -> void {
+                        Parse::Tree::PackagingNames import, bool fuzzing)
+    -> void {
   const auto& packaging = unit_info.parse_tree().packaging_decl();
 
   IdentifierId file_package_id =
       packaging ? packaging->names.package_id : IdentifierId::Invalid;
   const auto import_key = GetImportKey(unit_info, file_package_id, import);
   const auto& [import_package_name, import_library_name] = import_key;
 
-  // True if the import has `Main` as the package name, even if it comes from
-  // the file's packaging (diagnostics may differentiate).
-  bool is_explicit_main = import_package_name == MainPackageName;
-
-  bool is_cpp =
-      import_package_name == CppPackageName && !import_library_name.empty();
-
-  if (is_cpp) {
+  if (import_package_name == CppPackageName && !import_library_name.empty()) {
+    if (fuzzing) {
+      // Clang is not crash-resilient.
+      CARBON_DIAGNOSTIC(CppInteropFuzzing, Error,
+                        "Cpp import is not supported in fuzz testing");
```

Comment:
I think "Use "cannot" if needed, but try to use phrasing that doesn't require it. Avoid "allowed", "legal", "permitted", "valid", and related wording. For example:" applies here, from https://github.com/carbon-language/carbon-lang/blob/230a8ee598414334e224017e4d87c75061496046/toolchain/docs/diagnostics.md#diagnostic-message-style-guide

```suggestion
                        "Cpp import found during fuzzing");
```

Reply:
Thanks, done.